ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TipsLayout

```diff
@@ -0,0 +1,96 @@
+package org.cru.godtools.tutorial
```

Comment:
the filename should end with `.kt` to indicate that it's a Kotlin source file

Reply:
Just added it.